lib/src/generator/file_structure.dart

```diff
@@ -62,50 +114,29 @@ abstract class FileStructure {
   String get fileType;
 }
 
-class _FileStructureHtml implements FileStructure {
-  _FileStructureHtml(this.modelElement);
-
-  @override
-  // TODO: implement fileName
-  String get fileName => throw UnimplementedError();
-
-  @override
-  String get fileType => _html;
-
-  @override
-  // TODO: implement hasIndependentFile
-  bool get hasIndependentFile => throw UnimplementedError();
-
-  @override
-  // TODO: implement href
-  String get href => throw UnimplementedError();
-
+@visibleForTesting
+class FileStructureImpl implements FileStructure {
   @override
-  // TODO: implement htmlId
-  String get htmlId => throw UnimplementedError();
+  final String fileType;
+  String pathSafeName;
+  final String? kindAddition;
```

Comment:
Would you mind documenting what this field is?

Reply:
Not at all.  It's kind of hacky so definitely deserves some description.